docs/docs/development/local/index.md

```diff
@@ -11,11 +11,13 @@ to run a version of the site locally, at the following URLs:
 | *URL*                              | *Purpose*
 |------------------------------------| ------------------
 | https://performance.ddev.site      | [Drupal back end](../back-end)
-| https://performance.ddev.site:3000 | [Next.js front end](../front-end)
-| https://performance.ddev.site:3999 | Documentation
+| https://localhost:3000             | [Next.js front end](../front-end)
```

Comment:
This is tied up with the work I'm doing in https://github.com/CivicActions/pgov-cms/pull/74 -- Ideally it should stay performance.ddev.site unless you do some pro-maneuver of NOT using ddev.